spec/controllers/users_controller_spec.rb

```diff
@@ -78,22 +78,22 @@
       end
 
       it "contains the original author for reshares" do
-        FactoryGirl.create(:reshare, root: @sm, author: bob.person)
+        FactoryBot.create(:reshare, root: @sm, author: bob.person)
         get :public, params: {username: bob.username}, format: :atom
         doc = Nokogiri::XML(response.body)
         expect(doc.css("entry author name")[0].content).to eq(@sm.author_name)
       end
     end
 
     it 'includes reshares in the atom feed' do
-      reshare = FactoryGirl.create(:reshare, :author => @user.person)
+      reshare = FactoryBot.create(:reshare, :author => @user.person)
       get :public, params: {username: @user.username}, format: :atom
       expect(response.body).to include reshare.root.text
     end
 
     it 'do not show reshares in atom feed if origin post is deleted' do
-      post = FactoryGirl.create(:status_message, :public => true);
-      reshare = FactoryGirl.create(:reshare, :root => post, :author => @user.person)
+      post = FactoryBot.create(:status_message, :public => true);
+      reshare = FactoryBot.create(:reshare, :root => post, :author => @user.person)
```

Comment:
- Lint/UselessAssignment: Useless assignment to variable - `reshare`.
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.